bin/_test-helpers.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-# Override CI's `set -e` default, so we can catch errors manually and display
+# Override CIs' `set -e` default, so we can catch errors manually and display
```

Comment:
There are a number of ways to read what is intended here:

* {continuous integration} {possesses} {`set -e`}
* {continuous integration} {abbreviation marker} {`set -e`}
* {continuous integration} {plural marker} {`set -e`}

As this file is being used by multiple CI systems, I think the correct meaning is a cross between the first and the last, hence I'm moving the `'`.

Reply:
I think this is intended to be singular possessive so I think "CI's" is correct.